Lib/test/test_sysconfig.py

```diff
@@ -347,6 +352,21 @@ def test_get_platform(self):
 
         self.assertEqual(get_platform(), 'linux-i686')
 
+        # Android
+        os.name = 'posix'
+        sys.platform = 'android'
+        get_config_vars()['ANDROID_API_LEVEL'] = 9
+        for machine, abi in {
+            'x86_64': 'x86_64',
+            'i686': 'x86',
+            'aarch64': 'arm64_v8a',
+            'armv7l': 'armeabi_v7a',
```

Comment:
Is there a specific reason to use Android arch names, rather than what the rest of Python ecosystem uses?

(For reference: Fedora used their own arch names, and spent several years [migrating to the CPython ones](https://fedoraproject.org/wiki/Changes/Python_Upstream_Architecture_Names) when wheels became more universal. Android wheels will probably always be separate, but it seems they might be compared to other wheels more often than other Android apps.)